.github/workflows/storybook.yml

```diff
@@ -52,9 +52,8 @@ jobs:
           pnpm build-storybook
 
       - name: Deploy Storybook
-        uses: JamesIves/github-pages-deploy-action@3.6.2
+        uses: JamesIves/github-pages-deploy-action@v4
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The need to supply a GITHUB_TOKEN value was removed since v4.0.0. 
References:
[Migration Guide](https://github.com/JamesIves/github-pages-deploy-action/discussions/592)
[Commit history](https://github.com/JamesIves/github-pages-deploy-action/issues/525)